README.md

```diff
@@ -56,10 +71,154 @@ These three files can be downloaded directly from `inputs` and `dataset` directo
 
 Refer to [installation section](#installation) to set-up DeezyMatch on your local machine. 
 
-:warning: In the following tutorials, we assume a directory structure specified in the [this section](#data-and-directory-structure-in-tutorials).
+:warning: In the following tutorials, we assume a directory structure specified in [this section](#data-and-directory-structure-in-tutorials).
  
  Written in the Python programming language, DeezyMatch can be used as a stand-alone command-line tool or can be integrated as a module with other Python codes. In what follows, we describe DeezyMatch's functionalities in different examples and by providing both command lines and python modules syntaxes.
 
+### Quick tour 
+
+Refer to [installation section](#installation) to set-up DeezyMatch on your local machine. 
+
+:warning: In the following tutorials, we assume a directory structure specified in [this section](#data-and-directory-structure-in-tutorials).
+
+In this "quick tour", we go through all the DeezyMatch main functionalities with minimal explanations. If you want to know more about each module, refer to the relevant part of the README (also referenced in this section):
+
+* [Train a new model](#train-a-new-model):
+
+```python
+from DeezyMatch import train as dm_train
+
+# train a new model
+dm_train(input_file_path="./inputs/input_dfm.yaml", 
+         dataset_path="dataset/dataset-string-similarity_test.txt", 
+         model_name="test001")
+```
+
+* Plot the `log.txt` file (loss/accuracy/recall/F1-scores as a function of epoch):
+
+```python
+from DeezyMatch import plot_log
+
+# plot log file
+plot_log(path2log="./models/test001/log.txt", 
+         dataset="t001")
+```
+
+* [Finetune a pretrained model](#finetune-a-pretrained-model):
+
+```python
+from DeezyMatch import finetune as dm_finetune
+
+# fine-tune a pretrained model stored at pretrained_model_path and pretrained_vocab_path 
+dm_finetune(input_file_path="./inputs/input_dfm.yaml", 
+            dataset_path="dataset/dataset-string-similarity_test.txt", 
+            model_name="finetuned_test001",
+            pretrained_model_path="./models/test001/test001.model", 
+            pretrained_vocab_path="./models/test001/test001.vocab")
+```
```

Comment:
Hi @kasra-hosseini, I've just noticed that we don't say anywhere in the readme how to fine-tune on `n` number of rows on the Python interface (i.e. with `n_train_examples=xxx`). We should add it in the main `Finetune a pretrained model` section, I'm not sure we want to have it in the quick tour as well.

Reply:
Hi @mcollardanuy Thanks! I will add this to README (in the main section).

Reply:
Done, please take a look at: https://github.com/Living-with-machines/DeezyMatch/tree/feature/50-alias-detection-on-fly#finetune-a-pretrained-model